### ISSUE TITLE
[Bug]: <title>State value incorrect - divided by 1000 for Speedtest Ping value

### ISSUE THREAD

**Issue by Scoff123** [piitaya/lovelace-mushroom#1267]
### Requirements

- [X] I checked the troubleshooting section in the README to verify that I have the latest Mushroom version.
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Current Behavior

I am getting incorrect values displaying in my mushroom entity card for my speedtest ping result. When checking the actual state in Developer Tools in HA, the state correctly shows as 19 for example. However, the muchroom entity card shows a state of 0.019. The state always displays incorrectly by this factor. This was been happening for several weeks now, possibly as a result of an update somewhere. Previously, the state always used to display correctly. My other speedtest values for upload and download and both showing correctly, and they also use the mushroom entity card. 

The ping state shows correctly when using a regular entity card, so the problem seems to be related to the mushroom entity card. 

### Expected Behavior

Show the state of the entity exactly as it displays in the Developer Tools State display in HA

### Steps To Reproduce

Add a new mushroom entity card. Add entity 'sensor.speedtest_ping'. See incorrect state in card in primary information field. The same happens if you add the state to the Secondary Information field.

### Context

YAML state
```yaml

```

### Environment

```markdown
- Browser: Chrome
- HA Version: 2023.10.1
- Mushroom: 3.1.0
```

**Comment by piitaya** [comment]
Can you test with other official card (tile card for example). Do you have the same issue?

**Comment by Scoff123** [comment]
I did test with the built-in Entity card, and it does show correctly, so that pointed to the mushroom entity card as having the issue initially. I've now tried various other built-in cards, and some of them do also show the value incorrectly, so it's not isolated to the mushroom entity card after all. I guess I can close this now, and I'll have to dig deeper into whats causing this, as the state of the ping entity shows correctly in the HA Developer Tools State view. As the upload and download speeds are measured 'per second' I wonder if the ping is now also incorrectly being displayed in seconds, as opposed to milliseconds like it used to be, and how it should really be, to give the expected display. The ping entity unit of measurement is ms so that's what should be displayed. I guess maybe more of a generic frontend issue. Still not sure why the built-in entity card shows it correctly when other built-in cards don't though...

![image](https://github.com/piitaya/lovelace-mushroom/assets/96984460/389c56ac-119c-4635-8069-b7e721489183)

![image](https://github.com/piitaya/lovelace-mushroom/assets/96984460/28ee1017-ce37-4bee-8f60-20a40847ab14)
